core/mondoo-aws-security.mql.yaml

```
@@ -1994,7 +2077,8 @@ queries:
     title: Ensure Redshift clusters are not publicly accessible
     impact: 95
     filters: asset.platform == "aws-redshift-cluster"
-    mql: aws.redshift.cluster.publiclyAccessible == false
+    mql: |
+      aws.redshift.clusters.all(publiclyAccessible == false)
```

The check is on the `filters: asset.platform == "aws-redshift-cluster"`, which means we need to keep this syntax:

```suggestion
    mql: |
      aws.redshift.cluster.publiclyAccessible == false
```

---

Thank you. I've already deployed a variant version for this test. Meanwhile, investigations are ongoing regarding the presence of a Redshift cluster in the new AWS development environment. If it's confirmed that we're transitioning entirely to serverless, I'll remove the associated checks.